cmd/nuke.go

```diff
@@ -214,6 +226,15 @@ func (n *Nuke) Filter(item *Item) error {
 
 		match, err := filter.Match(prop)
 		if err != nil {
+			fferr, ok := err.(*config.FilterFailed)
+			if ok {
+				if fferr.Fatal {
+					return fferr.Err
+				}
+				item.State = ItemStateFilterFailed
+				item.Reason = fmt.Sprintf("couldn't determine if resource should be filtered: %v", fferr.Err)
+				return nil
+			}
```

Comment:
Decided on [this](https://go.dev/blog/error-handling-and-go) as the way I should handle errors